robottelo/host_helpers/cli_factory.py

```diff
@@ -726,13 +726,7 @@ def _setup_org_for_a_rh_repo(self, options=None):
         # If manifest does not exist, clone and upload it
         if len(self._satellite.cli.Subscription.exists({'organization-id': org_id})) == 0:
             with clone() as manifest:
-                self._satellite.put(manifest.path, manifest.name)
-            try:
-                self._satellite.cli.Subscription.upload(
-                    {'file': manifest.name, 'organization-id': org_id}
-                )
-            except CLIReturnCodeError as err:
-                raise CLIFactoryError(f'Failed to upload manifest\n{err.msg}') from err
+                self._satellite.upload_manifest(org_id, manifest.content)
```

Comment:
@lhellebr I missed this earlier, similar changes are required for `setup_org_for_a_rh_repo` helper, could you update it too?

Reply:
Can we leave it for another PR? This PR is part of CE effort so I'm just doing what is required for that, changing as little as possible to minimize possibility of breaking stuff.

Reply:
@lhellebr Sure, please make a note of it or create a issue for it for future, and when you raise a PR to fix it, let me know, thank you!